ui/src/pages/program/components/tabulka/ProgramTabulka.tsx

```diff
@@ -112,7 +111,7 @@ export const ProgramTabulka: FunctionComponent<ProgramTabulkaProps> = (
 
                     const časOdsazení = (hodinOd - posledníAktivitaDo + 24) % 24;
                     posledníAktivitaDo = hodinDo;
-                    const odsazení = range(časOdsazení).map(() => <td></td>);
+                    const odsazení = <td colSpan={časOdsazení}></td>;
```

Comment:
`web\soubory\blackarrow\program\program-trida.css:104` závisí na td elementech a že všechny prázdné mají přesnou šířku. Tohle stačí když bude zaručeno na th elementech nebo jinak aby každý sloupeček měl stejnou šířku. Myslím že jsem tam viděl někde min-width tak je to asi v pohodě